Fw/Cmd/README

```diff
@@ -3,5 +3,4 @@ CmdResponseAi.xml - XML definition of command response port, which is used to re
 CmdRegPortAi.xml - XML definition of a command registration port, which is used by components to register their set of commands
 CmdArgBuffer.hpp(.cpp) - C++ Definition of command argument buffer. Buffer is used to store serialized arguments for component commands
 CmdString.hpp(.cpp) - C++ Definition of a command string argument. Used by the code generator when a command has a string argument
-CmdPacket.hpp(.cpp) - C++ Definition of command packet. It is a derived class of ComPacket, which is the type used to send and receive data from ground or test software
-CmdModule.mdxml - MagicDraw project file describing command interface
\ No newline at end of file
+CmdPacket.hpp(.cpp) - C++ Definition of command packet. It is a derived class of ComPacket, which is the type used to send and receive data from ground or test software
```

Comment:
@LeStarch Do I have to add a new line at the end of each file?

Reply:
No, you should not need to.